packages/clickhouse-server.postinstall

```diff
@@ -26,6 +26,11 @@ if [ ! -f "/etc/debian_version" ]; then
 fi
 
 if [ "$1" = configure ] || [ -n "$not_deb_os" ]; then
+    installed=$(dpkg -l | grep "^ii\s\+clickhouse-server ")
```

Comment:
Interesting, what does `ii` and `+` mean? Is parsing the output of `dpkg` legitimate?

Reply:
**ii** for install and installed
>( **ic** for Installed but configuration incomplete)
> If it is the first installation, we might encounter the 'ic' (Incomplete) status when executing postinst script

\s\\+ for match multiple white spaces

Reply:
To avoid confusion, use single quotes in a shell script.